config/storybook/styles/_storybook-global.scss

```diff
@@ -43,3 +43,17 @@ h5 {
 h6 {
   @extend %text-xs;
 }
+
+// stylelint-disable selector-max-id
+.is-search-active .header,
+.is-search-active .image-lead,
+.is-search-active .pride-strip,
+.is-search-active .banner,
+.is-search-active .main,
+.is-search-active #main,
+.is-search-active .footer {
+  // TODO: figure out why we can't use @include mq(md) here
```

Comment:
Caused by the order of imports in storybook-app.scss.

```css
@import 'assets/styles/app';
@import './storybook-global';
```

Reply:
Sorted, worked fine with all of the extends ?!?!

Reply:
Doesn't matter with extends it seems, they can be defined anywhere. I didn't know that, but good to find out!